source/docs/api-reference/wpilib-integration/motorcontroller-integration.rst

```diff
@@ -0,0 +1,79 @@
+MotorController Integration
+===========================
+
+Phoenix Pro motor classes such as ``TalonFX`` (`Java <https://api.ctr-electronics.com/phoenixpro/release/java/com/ctre/phoenixpro/hardware/TalonFX.html>`__, `C++ <https://api.ctr-electronics.com/phoenixpro/release/cpp/classctre_1_1phoenixpro_1_1hardware_1_1_talon_f_x.html>`__) implement the ``MotorController`` (`Java <https://github.wpilib.org/allwpilib/docs/release/java/edu/wpi/first/wpilibj/motorcontrol/MotorController.html>`__, `C++ <https://github.wpilib.org/allwpilib/docs/release/cpp/classfrc_1_1_motor_controller.html>`__) interface.
+
+Phoenix Pro implements all inherited functions:
+
+- ``set``
+- ``setVoltage``
+- ``get``
+- ``setInverted``
+- ``getInverted``
+- ``disable``
+- ``stopMotor``
+
+Due to inheriting the ``MotorController`` interface, this allows motors such as ``TalonFX`` to be included as part of the ``DifferentialDrive`` constructor.
```

Comment:
1. Combine with first paragraph
    - Remove "Due to inheriting the ``MotorController`` interface,"
3. "this allows motors" -> "this allows motor controllers"